pylutron/__init__.py

```diff
@@ -931,7 +931,8 @@ class Event(LutronEvent):
 
     STATE_CHANGED: The button has been pressed.
         Params:
-          state: The boolean value of the new LED state.
+          state: The value of the new LED state.
+          0= off, 1= on, 2= 1 flash/sec, 3= 10 flash/sec
```

Comment:
Add class enums for OFF, ON, FLASH_SLOW, FLASH_FAST

Reply:
added